### DISCUSSION TITLE
Can Mitsuba3 render the point clouds with differentiable point positions？

### DISCUSSION THREAD

**Discussion by huohuohuohuohuohuohuohuo** [mitsuba-renderer/mitsuba3#774] [Q&A] [answered]
Can Mitsuba3 render the point clouds with differentiable point positions？

**Comment by njroussel** [comment, marked as answer]
Hi @huohuohuohuohuohuohuohuo 

No, currently Mitsuba doesn't have support for point clouds. The closest thing you could do is use many tiny `disk` plugins, however that could come with a very significant performance cost.
